### PR TITLE
CR-1131075 Support CU interrupt to APU

### PR DESCRIPTION
**Pull request by mamin506**
#### Problem solved by the commit
Support CU completion interrupt on APU

#### Risks (if any) associated the changes in the commit
No

#### What has been tested and how, request additional testing if necessary
vck5000 xbutil validate test, zcu104

#### Documentation impact (if any)
No

============================================
vck5000 xbutil validate iops result
```
$xbutil validate -d 04:00 -r iops
Starting validation for 1 devices

Validate Device           : [0000:04:00.1]
    Platform              : xilinx_vck5000_gen4x8_qdma_base_1
    SC Version            : 4.4.33
    Platform ID           : 2A6AB1C4-DFA1-27FD-7F24-E8F121039A2E
-------------------------------------------------------------------------------
Test 1 [0000:04:00.1]     : iops
    Details               : IOPS: 483580 (verify)
    Test Status           : [PASSED]
-------------------------------------------------------------------------------
Validation completed. Please run the command '--verbose' option for more details
```

### CODE REVIEW

**Comment by mamin506** [comment]
@stsoe , This is almost pure driver change. Maybe it is because `include/xgq_cmd_ert.h` is changed, so you are one of the reviewer.

**Comment by dayeh-xilinx** [comment]
retest this please. Known singletask.lock issue.

File: src/runtime_src/core/edge/drm/zocl/zocl_ctrl_ert.c [hunk 1]
```diff
@@ -518,6 +519,53 @@ static void zert_destroy_cu_xgqs(struct zocl_ctrl_ert *zert)
...
+
+	for (i = 0; i < num_irq; i++)
+		irqs[i] = of_irq_get(np, i);
+	ret = zocl_ert_create_intc(ZERT2DEV(zert), irqs, num_irq, 0, ERT_CU_INTC_DEV_NAME,
+				   &zert->zce_cu_intc);
+	if (ret)
+		zert_err(zert, "Failed to create CU intc device: %d", ret);
+
+out:
+	return 0;
```
Comment (maxzhen) [inline src/runtime_src/core/edge/drm/zocl/zocl_ctrl_ert.c:566]: missing kfree(irqs)?

**Review by maxzhen — CHANGES_REQUESTED** [review]
Please add AMD copyright to the changed file.

**Review by maxzhen — APPROVED** [review]

**Comment by dayeh-xilinx** [comment]
retest this please. one of EDGE_HW tests time out.

**Review by chvamshi-xilinx — APPROVED** [review]

**Pull request merged**